cloudinit/distros/__init__.py

```diff
@@ -118,6 +120,7 @@ def __init__(self, name, cfg, paths):
         self.name = name
         self.networking: Networking = self.networking_cls()
         self.dhcp_client_priority = [dhcp.IscDhclient, dhcp.Dhcpcd]
+        self.net_ops = iproute2.Iproute2
```

Comment:
is this intended to be configurable?  defined as class variable and instance variable

Reply:
that's a question for @holmanb

Reply:
> is this intended to be configurable? defined as class variable and instance variable

This is not intended to be configurable at runtime. This is defined as an instance variable to avoid unpickle issues across the upgrade path. It might not be required as a class variable, I don't recall why it is defined both ways.